kratos/python/add_properties_to_python.cpp

```diff
@@ -147,10 +149,37 @@ bool HasTableHelperFunction1( TContainerType& rContainer,
     return rContainer.HasTable(XVar, YVar);
 }
 
+template<typename TVariableType>
+void AddInterfaceToAccessorFold(pybind11::class_<Properties, Properties::Pointer, Properties::BaseType>& pyProperties) {
+
+    pyProperties
+    .def("GetAccessor", [](Properties &rProperties, Variable<TVariableType> &rVariable) { 
+            auto accessor = &rProperties.pGetAccessor(rVariable);
+            
+            KRATOS_ERROR_IF(*accessor == nullptr) << "Trying to get a consumed or invalid Accessor." << std::endl;
+                
+            return accessor; 
+        }, py::return_value_policy::reference_internal)
+    .def("SetAccessor", [](Properties &rProperties, Variable<TVariableType> &rVariable, std::unique_ptr<Accessor>& rpAccessor) {
+            if (rpAccessor == nullptr)
+                KRATOS_ERROR << "Trying to set a consumed or invalid Accessor. Accessors are unique. Please create a different one." << std::endl;
+
+            rProperties.SetAccessor(rVariable, std::move(rpAccessor));
```

Comment:
i think you should set rpAccessor to a nullptr explicilty, otherwise what is left on rpAccessor is undefined

Reply:
I cannot change rpAccessor because its a refference pointer to the python wrapper. I can only checks that points to nullptr because of either the move ir or in another place.

Reply:
well, then you have no guarantees about the status in which it will be left after moving it (as i recall it is undefined behaviour) ... this may be a problem since you are checking for multiple usage

Reply:
The I don't know how yo check the correctness of this :S